main.py

```diff
@@ -90,10 +87,10 @@ def find_offers(self):
             time.sleep(2)
 
             # find the last page and construct url of each page based on the total amount of pages
-            find_pages = self.driver.find_elements_by_class_name("artdeco-pagination__indicator.artdeco-pagination__indicator--number")
+            find_pages = self.driver.find_elements(By.CLASS_NAME, "artdeco-pagination__indicator.artdeco-pagination__indicator--number.ember-view")
             total_pages = find_pages[len(find_pages)-1].text
             total_pages_int = int(re.sub(r"[^\d.]", "", total_pages))
-            get_last_page = self.driver.find_element_by_xpath("//button[@aria-label='Page "+str(total_pages_int)+"']")
+            get_last_page = self.driver.find_element(By.XPATH, "//button[@aria-label='Page "+str(total_pages_int)+"']")
```

Comment:
Exception has occurred: NoSuchElementException
Message: no such element: Unable to locate element: {"method":"xpath","selector":"//button[@aria-label='Page ']"}